site.gradle

```diff
@@ -0,0 +1,63 @@
+plugins {
```

Comment:
If we have a site.gradle file, we don't need site.xml anymore, right? I think this PR should remove site.xml